### ISSUE TITLE
Loss = inf after 5 epochs

### ISSUE THREAD

**Issue by mgfcm** [buriburisuri/speech-to-text-wavenet#61]
![screenshot_2017-04-07_09-37-19](https://cloud.githubusercontent.com/assets/9610272/24782000/0a55fda0-1b76-11e7-9e23-8e3b01e25c2b.png)

Trying to reproduce the results. Currently training on the default settings on a single Titan X. Is this normal for the first few epochs?

**Comment by zessi1990** [comment]
my loss  Epoch[050:gs=0] - loss = NA  when i  use the only  cpu

**Comment by buriburisuri** [comment]
@mgfcm, @zessi1990  Yes, it's okay if the loss in the tensorboard is decreasing.  This problem may be occurred by the CTC loss in the tensorflow.

**Comment by allipilli-harshitha** [comment]
hi @buriburisuri I got the same error for the first epoch i.e., loss =      inf. what could be the reason for this infinity loss. could you please suggest what need to be changed.
initially my training has stopped in the first epoch after 92%. so i changed few parameters in sg_ctc. preprocess_collapse_repeated=True and ignore_longer_outputs_than_inputs=True.
After changing these parameters, training for epoch 1 is 100% but loss is inf.
